docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: "3.9"
```

Comment:
3.9 is not listed at https://docs.docker.com/compose/compose-file/compose-versioning/ and my fairly recent docker v19.03.13-beta2 
 / docker-compose 1.26.2 errors on it.